src/providers/offline-content.ts

```diff
@@ -86,6 +86,19 @@ export class CoreOfflineAuthProvider {
             }
         });
     }
-};
 
-export class CoreOfflineAuth extends makeSingleton(CoreOfflineAuthProvider) {}
\ No newline at end of file
+    async getHashedCredentials(siteId: string): Promise<string> {
+        await this.dbReady;
+
+        try {
+            const entry = await this.appDB.getRecord(this.OFFLINE_AUTH_TABLE, {siteId: siteId});
```

Comment:
wouldn't change but you can shorten to `getRecord(this.OFFLINE_AUTH_TABLE, {siteId});`

Reply:
Because the key is the same as the value

Reply:
oh interesting